app/views/naming/_show.html.erb

```diff
@@ -36,19 +42,27 @@
             </td>
             <td>
               <% txt = naming.vote_percent.round.to_s.html_safe + "%" %>
-              <%= link_with_query(h(txt), {controller: :observer, action: :show_votes, id: naming.id},
-                                  {data: {role: "open_popup", id: naming.id}}) if naming.votes.any? %>
+              <%= link_with_query(h(txt), { controller: :observer,
+                                            action: :show_votes,
+                                            id: naming.id },
+                                  { data: { role: "open_popup",
+                                            id: naming.id } }
+                  ) if naming.votes.any? %>
               <%= "(#{naming.votes.any? == 0 ? :show_namings_no_votes.t : naming.votes.length})" %>
             </td>
             <td>
               <% if logged_in %>
                 <% @vote = @votes[naming.id]
-                menu = Vote.confidence_menu
-                if check_permission(naming) ? (!@vote || @vote.value == 0) : true
-                  menu = [Vote.no_opinion] + menu
+                   menu = Vote.confidence_menu
```

Comment:
I think github is indicating that there are tabs rather than spaces on these three lines.